app/controllers/session_answers_controller.rb

```diff
@@ -10,8 +12,24 @@ def update
     redirect_to session_flow_path(id: params[:id], node_name: next_node_name)
   end
 
+  def destroy
+    session.delete(:responses)
+
+    if params[:ext_r] == "true"
+      redirect_to "https://bbc.co.uk/news"
+    else
+      redirect_to "/#{params[:id]}"
+    end
+  end
+
 private
 
+  def set_cache_headers
+    response.headers["Cache-Control"] = "no-cache, no-store"
+    response.headers["Pragma"] = "no-cache"
+    response.headers["Expires"] = "Mon, 01 Jan 1990 00:00:00 GMT"
+  end
```

Comment:
Can we have tests that the cache headers are being set?

Reply:
Hi @theseanything - thanks.  Tests added.